02-Advanced-topics/05-GSD-visualization.ipynb

```diff
@@ -0,0 +1,242 @@
+{
```

Comment:
<p>Are fully unwrapped coords the typical way you look at these types of simulations?</p><p> </p><p>I think it is too much for a small example to wrap connected molecules back into the primary image. But we could consider just deleting bonds that are longer than <em>L/2</em> or those that connect particles of different images.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/fresnel-examples/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>This is a good question... fully unwrapped coords isn't the normal way I look at these. I'll play around a bit to find a compromise between keeping things simple, but also good looking (right now it looks more like confetti exploding out of a box) </p>

Reply:
Okay bond's removed that span greater than L/2